install/README.md

```diff
@@ -0,0 +1,14 @@
+# Install tools and workflows on a Galaxy instance
+
+
```

Comment:
can you mention that you need ephemeris please ... pip install ...

Reply:
And a short introduction ... given ephemeris and the following files you can install all needed tools into any given Galaxy instance which you have admin access to ...

Reply:
I'm on it ;)